src/main/java/ua/lviv/iot/parkingServer/controller/VehicleController.kt

```diff
@@ -29,5 +29,5 @@ class VehicleController(private val vehicleService: VehicleServiceInterface) {
         vehicleService.updateEntity(id, vehicle)
 
     @DeleteMapping("/{id}")
-    fun deleteVehicle(@PathVariable id: Long): Vehicle = vehicleService.deleteEntity(id)
+    fun deleteVehicle(@PathVariable id: Long) = vehicleService.deleteEntity(id)
```

Comment:
It was clear what the function returned, but now it doesn't :(

Please fix all the places

Reply:
✅